Cython/Compiler/ModuleNode.py

```diff
@@ -3120,6 +3157,11 @@ def generate_type_import_call(self, type, code, import_generator, error_code=Non
     def generate_type_ready_code(self, entry, code):
         Nodes.CClassDefNode.generate_type_ready_code(entry, code)
 
+    def module_is_main_name(self):
```

Comment:
I was failing to understand the meaning of this function by its name several times further up, before I found it here and investigated what it does. Seems difficult to name it properly, but a better name might be `is_main_module_flag_cname()`.